libqtile/widget/windowname.py

```diff
@@ -52,6 +54,12 @@ def on_screen_changed():
             if self.for_current_screen:
                 self.update()
 
+    def truncate(self, text, char_limit):
```

Comment:
There's no need to pass `char_limit` through if it is available at `self.char_limit`